addons/auto-follow-topics/addon.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Auto follow topics",
```

Comment:
Just wondering if we can think of a better name. Ideally it should imply that the "auto follow" happens after posting.

Reply:
I'm thinking of removing the word "auto" and keeping it simple: `Follow topics after posting`

Reply:
> I'm thinking of removing the word "auto" and keeping it simple: `Follow topics after posting`

How about `Automatically follow topics after posting`? It feels more automated that way.

Reply:
> > I'm thinking of removing the word "auto" and keeping it simple: `Follow topics after posting`
> 
> How about `Automatically follow topics after posting`? It feels more automated that way.

But that's a mouthful.

Reply:
The word "automatically" can stay as the first word of the description.